platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/MigrationCommands.java

```diff
@@ -27,6 +36,38 @@
 
     public static final String NAMESPACE = "migration";
 
+    @Reference
+    protected ConfigurationMigrationService configurationMigrationService;
+
+    protected Security security = Security.getInstance();
+
+    protected Path defaultExportDirectory = Paths.get(System.getProperty("ddf.home"), "exported");
+
+    public void setDefaultExportDirectory(Path path) {
+        this.defaultExportDirectory = path;
+    }
+
+    public void setSecurity(Security security) {
```

Comment:
❓ If this is here only for unit testing, can we make it package-private, move it down and add the `VisibleForTesting` annotation?

Reply:
done